NPM updates removing "@" from scoped packages

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

https://gitlab.com/renovate-bot/renovate-runner/-/blob/main/templates/renovate.gitlab-ci.yml

### Please select which platform you are using if self-hosting.

gitlab.com

### If you're self-hosting Renovate, tell us what version of the platform you run.

gitlab.com

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the bug

When updating NPM modules the `@` is being removed from all scoped package names (package references and URLs) in package-lock.json. The expected update, if not a scoped package, appears to be made correctly, but the package-lock.json is obviously broken in this state.

For example:

```json
    "node_modules/@babel/code-frame": {
      "version": "7.16.7",
      "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.16.7.tgz",
      "integrity": "sha512-iAXqUn8IIeBTNd72xsFlgaXHkMBMt6y4HJp1tIaK465CWLT/fG1aqB7ykr95gHHmlBdGbFeWWfyB4NJJ0nmeIg==",
      "dev": true,
      "dependencies": {
        "@babel/highlight": "^7.16.7"
      },
      "engines": {
        "node": ">=6.9.0"
      }
    },
```

becomes:

```json
    "node_modules/babel/code-frame": {
      "version": "7.16.7",
      "resolved": "https://registry.npmjs.org/babel/code-frame/-/code-frame-7.16.7.tgz",
      "integrity": "sha512-iAXqUn8IIeBTNd72xsFlgaXHkMBMt6y4HJp1tIaK465CWLT/fG1aqB7ykr95gHHmlBdGbFeWWfyB4NJJ0nmeIg==",
      "dev": true,
      "dependencies": {
        "babel/highlight": "^7.16.7"
      },
      "engines": {
        "node": ">=6.9.0"
      }
    },
```

No obvious errors were seen in the logs.

There is a minimum viable reproduction at https://gitlab.com/aarongoldenthal/renovate-npm-scoped-packages/-/merge_requests/3/diffs.

### Relevant debug logs

<details><summary>Logs</summary>

```
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"Matched 1 file(s) for manager npm: package.json","time":"2022-04-21T01:13:36.161Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"npm file package.json has name \"renovate-npm-scoped-packages\"","time":"2022-04-21T01:13:36.163Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"Detecting pnpm Workspaces","time":"2022-04-21T01:13:36.166Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"Detecting Lerna and Yarn Workspaces","time":"2022-04-21T01:13:36.166Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"Finding locked versions","time":"2022-04-21T01:13:36.167Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"Found package-lock.json for package.json","time":"2022-04-21T01:13:36.167Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"Found npm package files","time":"2022-04-21T01:13:36.168Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"Found 1 package file(s)","time":"2022-04-21T01:13:36.168Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":30,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","baseBranch":"main","stats":{"managers":{"npm":{"fileCount":1,"depCount":2}},"total":{"fileCount":1,"depCount":2}},"msg":"Dependency extraction complete","time":"2022-04-21T01:13:36.169Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","baseBranch":"main","msg":"Package releases lookups complete","time":"2022-04-21T01:13:36.349Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"branchifyUpgrades","time":"2022-04-21T01:13:36.349Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"1 flattened updates found: jest-junit","time":"2022-04-21T01:13:36.387Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"Returning 1 branch(es)","time":"2022-04-21T01:13:36.387Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"Fetching changelog: https://github.com/jest-community/jest-junit (13.1.0 -> 13.2.0)","time":"2022-04-21T01:13:36.388Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"config.repoIsOnboarded=true","time":"2022-04-21T01:13:37.119Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","config":{"npm":[{"packageFile":"package.json","deps":[{"depType":"devDependencies","depName":"@jest/globals","currentValue":"27.5.1","datasource":"npm","prettyDepType":"devDependency","lockedVersion":"27.5.1","depIndex":0,"updates":[],"warnings":[],"versioning":"npm","sourceUrl":"https://github.com/facebook/jest","sourceDirectory":"packages/jest-globals","currentVersion":"27.5.1","fixedVersion":"27.5.1"},{"depType":"devDependencies","depName":"jest-junit","currentValue":"13.1.0","datasource":"npm","prettyDepType":"devDependency","lockedVersion":"13.1.0","depIndex":1,"updates":[{"bucket":"non-major","newVersion":"13.2.0","newValue":"13.2.0","releaseTimestamp":"2022-04-20T22:13:43.698Z","newMajor":13,"newMinor":2,"updateType":"minor","branchName":"renovate/jest-junit-13.x"}],"warnings":[],"versioning":"npm","sourceUrl":"https://github.com/jest-community/jest-junit","currentVersion":"13.1.0","isSingleVersion":true,"fixedVersion":"13.1.0"}],"packageJsonName":"renovate-npm-scoped-packages","packageFileVersion":"1.0.0","packageJsonType":"app","npmLock":"package-lock.json","managerData":{"yarnZeroInstall":false},"skipInstalls":true,"constraints":{},"lockFiles":["package-lock.json"]}]},"msg":"packageFiles with updates","time":"2022-04-21T01:13:37.120Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"processRepo()","time":"2022-04-21T01:13:37.121Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"Processing 1 branch: renovate/jest-junit-13.x","time":"2022-04-21T01:13:37.121Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"Calculating hourly PRs remaining","time":"2022-04-21T01:13:37.121Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"currentHourStart=2022-04-21T01:00:00.000+00:00","time":"2022-04-21T01:13:37.396Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"PR hourly limit remaining: 2","time":"2022-04-21T01:13:37.396Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"Calculating prConcurrentLimit (10)","time":"2022-04-21T01:13:37.397Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"getBranchPr(renovate/jest-junit-13.x)","time":"2022-04-21T01:13:37.397Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"findPr(renovate/jest-junit-13.x, undefined, open)","time":"2022-04-21T01:13:37.397Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"getPr(3)","time":"2022-04-21T01:13:37.397Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"getMR(3)","time":"2022-04-21T01:13:37.397Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"1 PRs are currently open","time":"2022-04-21T01:13:37.668Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"PR concurrent limit remaining: 9","time":"2022-04-21T01:13:37.668Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","prsRemaining":2,"msg":"Calculated maximum PRs remaining this run","time":"2022-04-21T01:13:37.668Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"PullRequests limit = 2","time":"2022-04-21T01:13:37.668Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"Calculating hourly PRs remaining","time":"2022-04-21T01:13:37.668Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"currentHourStart=2022-04-21T01:00:00.000+00:00","time":"2022-04-21T01:13:37.669Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"PR hourly limit remaining: 2","time":"2022-04-21T01:13:37.669Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"Calculating branchConcurrentLimit (10)","time":"2022-04-21T01:13:37.669Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"1 already existing branches found: renovate/jest-junit-13.x","time":"2022-04-21T01:13:37.669Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"Branch concurrent limit remaining: 9","time":"2022-04-21T01:13:37.669Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branchesRemaining":2,"msg":"Calculated maximum branches remaining this run","time":"2022-04-21T01:13:37.669Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"Branches limit = 2","time":"2022-04-21T01:13:37.669Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"Setting current branch to main","time":"2022-04-21T01:13:37.670Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","branchName":"main","latestCommitDate":"2022-04-20T20:11:28-05:00","msg":"latest commit","time":"2022-04-21T01:13:37.705Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"getBranchPr(renovate/jest-junit-13.x)","time":"2022-04-21T01:13:37.714Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"findPr(renovate/jest-junit-13.x, undefined, open)","time":"2022-04-21T01:13:37.715Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"getPr(3)","time":"2022-04-21T01:13:37.715Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"getMR(3)","time":"2022-04-21T01:13:37.715Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"branchExists=true","time":"2022-04-21T01:13:37.716Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"dependencyDashboardCheck=undefined","time":"2022-04-21T01:13:37.716Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"PR rebase requested=true","time":"2022-04-21T01:13:37.716Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"Checking if PR has been edited","time":"2022-04-21T01:13:37.721Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","branchName":"renovate/jest-junit-13.x","msg":"Branch has not been modified","time":"2022-04-21T01:13:37.747Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"Found existing branch PR","time":"2022-04-21T01:13:37.747Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"Checking schedule(at any time, null)","time":"2022-04-21T01:13:37.747Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"No schedule defined","time":"2022-04-21T01:13:37.747Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"Manual rebase requested via Dependency Dashboard","time":"2022-04-21T01:13:37.748Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"Using reuseExistingBranch: false","time":"2022-04-21T01:13:37.748Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"manager.getUpdatedPackageFiles() reuseExistinbranch=false","time":"2022-04-21T01:13:37.748Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"npm.updateDependency(): devDependencies.jest-junit = 13.2.0","time":"2022-04-21T01:13:37.770Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"Updating jest-junit in package.json","time":"2022-04-21T01:13:37.771Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"Updated 1 package files","time":"2022-04-21T01:13:37.771Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"Getting updated lock files","time":"2022-04-21T01:13:37.772Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","packageFiles":["package.json"],"msg":"Writing package.json files","time":"2022-04-21T01:13:37.773Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"Writing package-lock.json","time":"2022-04-21T01:13:37.773Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"Massaging npm lock file before writing to disk","time":"2022-04-21T01:13:37.785Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"Writing any updated package files","time":"2022-04-21T01:13:37.786Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"Writing package.json","time":"2022-04-21T01:13:37.786Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"No npmrc file found in repository","time":"2022-04-21T01:13:37.789Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"Writing updated .npmrc file to /builds/aarongoldenthal/renovate/renovate/repos/gitlab/aarongoldenthal/renovate-npm-scoped-packages/.npmrc","time":"2022-04-21T01:13:37.789Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"Generating package-lock.json for .","time":"2022-04-21T01:13:37.789Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"Spawning npm install to create /builds/aarongoldenthal/renovate/renovate/repos/gitlab/aarongoldenthal/renovate-npm-scoped-packages/package-lock.json","time":"2022-04-21T01:13:37.790Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"Updating lock file only","time":"2022-04-21T01:13:37.790Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"No node constraint found - using latest","time":"2022-04-21T01:13:37.790Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","image":"node","msg":"Using docker to execute","time":"2022-04-21T01:13:37.791Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","toolName":"npm","resolvedVersion":"8.7.0","msg":"Resolved version","time":"2022-04-21T01:13:40.194Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","image":"docker.io/renovate/node","msg":"No tag or tagConstraint specified","time":"2022-04-21T01:13:40.194Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"Fetching Docker image: docker.io/renovate/node","time":"2022-04-21T01:13:40.195Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"Finished fetching Docker image","time":"2022-04-21T01:13:48.497Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","command":"docker run --rm --name=renovate_node --label=renovate_child -v \"/builds/aarongoldenthal/renovate/renovate/repos/gitlab/aarongoldenthal/renovate-npm-scoped-packages\":\"/builds/aarongoldenthal/renovate/renovate/repos/gitlab/aarongoldenthal/renovate-npm-scoped-packages\" -v \"/builds/aarongoldenthal/renovate/renovate/cache\":\"/builds/aarongoldenthal/renovate/renovate/cache\" -e NPM_CONFIG_CACHE -e npm_config_store -w \"/builds/aarongoldenthal/renovate/renovate/repos/gitlab/aarongoldenthal/renovate-npm-scoped-packages\" docker.io/renovate/node bash -l -c \"install-tool npm 8.7.0 && hash -d npm 2>/dev/null || true && npm install --package-lock-only --no-audit --ignore-scripts\"","msg":"Executing command","time":"2022-04-21T01:13:48.578Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","cmd":"docker run --rm --name=renovate_node --label=renovate_child -v \"/builds/aarongoldenthal/renovate/renovate/repos/gitlab/aarongoldenthal/renovate-npm-scoped-packages\":\"/builds/aarongoldenthal/renovate/renovate/repos/gitlab/aarongoldenthal/renovate-npm-scoped-packages\" -v \"/builds/aarongoldenthal/renovate/renovate/cache\":\"/builds/aarongoldenthal/renovate/renovate/cache\" -e NPM_CONFIG_CACHE -e npm_config_store -w \"/builds/aarongoldenthal/renovate/renovate/repos/gitlab/aarongoldenthal/renovate-npm-scoped-packages\" docker.io/renovate/node bash -l -c \"install-tool npm 8.7.0 && hash -d npm 2>/dev/null || true && npm install --package-lock-only --no-audit --ignore-scripts\"","durationMs":7242,"stdout":"Installing legacy tool npm v8.7.0\n\nadded 1 package in 2s\n8.7.0\n\nup to date in 733ms\n","stderr":"npm notice \nnpm notice New minor version of npm available! 8.5.0 -> 8.7.0\nnpm notice Changelog: <https://github.com/npm/cli/releases/tag/v8.7.0>\nnpm notice Run `npm install -g npm@8.7.0` to update!\nnpm notice \n","msg":"exec completed","time":"2022-04-21T01:13:55.740Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"package-lock.json needs updating","time":"2022-04-21T01:13:55.754Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","updatedArtifacts":["package-lock.json"],"msg":"Updated 1 lock files","time":"2022-04-21T01:13:55.755Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"Ensuring comment \"⚠ Artifact update problem\" in #3 is removed","time":"2022-04-21T01:13:55.755Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"Getting comments for #3","time":"2022-04-21T01:13:55.756Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"Found 8 comments","time":"2022-04-21T01:13:55.996Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"isBranchConflicted(main, renovate/jest-junit-13.x)","time":"2022-04-21T01:13:55.997Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","gitAuthorName":"GoldenBot","msg":"Setting git author name","time":"2022-04-21T01:13:55.997Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","gitAuthorEmail":"bot@aarongoldenthal.com","msg":"Setting git author email","time":"2022-04-21T01:13:56.062Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"2 file(s) to commit","time":"2022-04-21T01:13:57.029Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"Preparing files for committing to branch renovate/jest-junit-13.x","time":"2022-04-21T01:13:57.029Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","deletedFiles":[],"ignoredFiles":[],"result":{"author":null,"branch":"renovate/jest-junit-13.x","commit":"895f16313b72beb9079d7052ef56a00e1ba676fb","root":false,"summary":{"changes":2,"insertions":132,"deletions":132}},"msg":"git commit","time":"2022-04-21T01:13:57.267Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"Pushing branch renovate/jest-junit-13.x","time":"2022-04-21T01:13:57.280Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","result":{"pushed":[],"ref":{"local":"refs/remotes/origin/renovate/jest-junit-13.x"},"remoteMessages":{"all":["View merge request for renovate/jest-junit-13.x:","https://gitlab.com/aarongoldenthal/renovate-npm-scoped-packages/-/merge_requests/3"]}},"msg":"git push","time":"2022-04-21T01:13:58.451Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":30,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","commitSha":"895f16313b72beb9079d7052ef56a00e1ba676fb","msg":"Branch updated","time":"2022-04-21T01:13:58.452Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"Checking if we can automerge branch","time":"2022-04-21T01:13:58.453Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"mergeStatus=no automerge","time":"2022-04-21T01:13:58.453Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"Ensuring PR","time":"2022-04-21T01:13:58.453Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"There are 0 errors and 0 warnings","time":"2022-04-21T01:13:58.453Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"getBranchPr(renovate/jest-junit-13.x)","time":"2022-04-21T01:13:58.454Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"findPr(renovate/jest-junit-13.x, undefined, open)","time":"2022-04-21T01:13:58.454Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"getPr(3)","time":"2022-04-21T01:13:58.454Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"getMR(3)","time":"2022-04-21T01:13:58.455Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"Found existing PR","time":"2022-04-21T01:13:58.456Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"Processing existing PR","time":"2022-04-21T01:13:58.509Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":30,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","pr":3,"prTitle":"Update dependency jest-junit to v13.2.0","msg":"PR updated","time":"2022-04-21T01:13:58.931Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branch":"renovate/jest-junit-13.x","msg":"PR is not configured for automerge","time":"2022-04-21T01:13:58.931Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"getBranchPr(renovate/jest-junit-13.x)","time":"2022-04-21T01:13:58.945Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"findPr(renovate/jest-junit-13.x, undefined, open)","time":"2022-04-21T01:13:58.945Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"getPr(3)","time":"2022-04-21T01:13:58.945Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"getMR(3)","time":"2022-04-21T01:13:58.945Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"Ensuring Dependency Dashboard","time":"2022-04-21T01:13:58.947Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"ensureIssue()","time":"2022-04-21T01:13:59.141Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"Updating issue","time":"2022-04-21T01:13:59.142Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"Removing any stale branches","time":"2022-04-21T01:13:59.447Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"config.repoIsOnboarded=true","time":"2022-04-21T01:13:59.447Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","branchList":["renovate/jest-junit-13.x"],"renovateBranches":["renovate/jest-junit-13.x"],"msg":"Branch lists","time":"2022-04-21T01:13:59.447Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"remainingBranches=","time":"2022-04-21T01:13:59.447Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"No branches to clean up","time":"2022-04-21T01:13:59.448Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","msg":"ensureIssueClosing()","time":"2022-04-21T01:13:59.448Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","splits":{"init":3011,"extract":593,"lookup":951,"update":21812},"total":26891,"msg":"Repository timing splits (milliseconds)","time":"2022-04-21T01:13:59.456Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":20,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","urls":{"https://api.github.com/repos/jest-community/jest-junit (GET,200)":1,"https://api.github.com/repos/jest-community/jest-junit/git/trees/master (GET,200)":1,"https://api.github.com/repos/jest-community/jest-junit/releases (GET,200)":1,"https://api.github.com/repos/jest-community/jest-junit/tags (GET,200)":1,"https://api.github.com/repos/whitesource/merge-confidence/contents/beta.json (GET,200)":1,"https://gitlab.com/api/v4/projects/aarongoldenthal%2Frenovate-npm-scoped-packages/issues (GET,200)":1,"https://gitlab.com/api/v4/projects/aarongoldenthal%2Frenovate-npm-scoped-packages/issues/1 (GET,200)":2,"https://gitlab.com/api/v4/projects/aarongoldenthal%2Frenovate-npm-scoped-packages/issues/1 (PUT,200)":1,"https://gitlab.com/api/v4/projects/aarongoldenthal%2Frenovate-npm-scoped-packages/merge_requests (GET,200)":1,"https://gitlab.com/api/v4/projects/aarongoldenthal%2Frenovate-npm-scoped-packages/merge_requests/3 (GET,200)":1,"https://gitlab.com/api/v4/projects/aarongoldenthal%2Frenovate-npm-scoped-packages/merge_requests/3 (PUT,200)":1,"https://gitlab.com/api/v4/projects/aarongoldenthal%2Frenovate-npm-scoped-packages/merge_requests/3/notes (GET,200)":1,"https://registry.npmjs.org/@jest%2Fglobals (GET,200)":1,"https://registry.npmjs.org/jest-junit (GET,200)":1,"https://registry.npmjs.org/npm (GET,200)":1},"hostStats":{"api.github.com":{"requestCount":5,"requestAvgMs":155,"queueAvgMs":0},"gitlab.com":{"requestCount":8,"requestAvgMs":257,"queueAvgMs":0},"registry.npmjs.org":{"requestCount":3,"requestAvgMs":426,"queueAvgMs":0}},"totalRequests":16,"msg":"http statistics","time":"2022-04-21T01:13:59.467Z","v":0}
{"name":"renovate","hostname":"runner--azerasq-project-28849687-concurrent-0","pid":13,"level":30,"logContext":"gWxsA-QNLTa3ZJyzZgUBZ","repository":"aarongoldenthal/renovate-npm-scoped-packages","durationMs":26891,"msg":"Repository finished","time":"2022-04-21T01:13:59.467Z","v":0}
```

</details>


### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction repository in the bug description

Hi guys,

I'm seeing this too. Starting with version 32.26.4 of Renovate all the Merge Requests opened by Renovate are failing with errors like the ones below

```
node:internal/modules/cjs/loader:936
  throw err;
  ^
Error: Cannot find module '@istanbuljs/schema'
```

```
node:internal/modules/cjs/loader:936
  throw err;
  ^
Error: Cannot find module '@colors/colors/safe'
```

All @scoped packages.

Thanks,
David

--- a/lib/modules/manager/npm/post-update/index.ts
+++ b/lib/modules/manager/npm/post-update/index.ts
@@ -526,17 +526,17 @@ export async function getAdditionalFiles(
     NODE_ENV: 'dev',
   };
 
-  let token = '';
+  let token: string | undefined;
   try {
-    ({ token = '' } = hostRules.find({
+    ({ token } = hostRules.find({
       hostType: config.platform,
       url: 'https://api.github.com/',
     }));
-    token += '@';
+    token = token ? `${token}@` : token;
   } catch (err) {
     logger.warn({ err }, 'Error getting token for packageFile');
   }
-  const tokenRe = regEx(`${token}`, 'g', false);
+  const tokenRe = regEx(`${token ?? ''}`, 'g', false);
   const localDir = GlobalConfig.get('localDir')!;
   for (const npmLock of dirs.npmLockDirs) {
     const lockFileDir = upath.dirname(npmLock);

